string.go

```diff
@@ -33,6 +37,12 @@ func sanitizeStrField(s Sanitizer, structValue reflect.Value, idx int) error {
 	}
 
 	for _, field := range fields {
+		if !field.CanSet() {
+			// private/unexported field which cannot be changed, no point trying to sanitize it.
+			// we will not be able to write it using field.SetString (it will panic while checking whether it is assignable)
+			return nil
```

Comment:
should it be instead:
```suggestion
			continue
```

or alternatively move outside of loop

Reply:
You are right, it should be `continue` . This will check each field of struct ( in loop ) and ignore `unexported`